### DISCUSSION TITLE
Update the documentation to point to this discussion

### DISCUSSION THREAD

**Discussion by RichMacDonald** [mapstruct/mapstruct#3195] [Ideas]
In the [github home page](https://github.com/mapstruct/mapstruct), in the "Documentation and getting help" chapter, the "come and join the mapstruct-users group" hyperlink points to the [google groups page](https://groups.google.com/forum/?hl=en#!forum/mapstruct-users).

That page is not monitored.

Suggest (1) Adding a post to the google groups page to point to the new discussion area, and (2) updating the github home page as well.

**Comment by filiphr** [comment]
Thanks a lot for this @RichMacDonald. I've applied your suggestions:

1) The description of the google groups is changed to point to the GitHub Discussions
2) I've updated the Readme to point to the discussions (https://github.com/mapstruct/mapstruct/commit/162fdb44f402e47d549778d57f369db144ce5e0d)
